vscode-extension/syntaxes/rascal.tmLanguage.json

```diff
@@ -1,351 +1,549 @@
+      "name": "/inner/single/literal.regExp",
+      "captures": {
+        "1": {
+          "name": "constant.regexp"
+        },
+        "2": {
+          "name": ""
```

Comment:
@sungshik is this a bug?

Reply:
Nope, but it's not particularly elegant either 😉 Some generated regexes have groups that are used only internally for matching, without a category. These show up here as empty strings. This has no impact on the highlighting, but I agree it would be nicer to not have these degenerate entries in the `captures` map at all. It requires the addition of just a simple non-emptiness check. I've fixed that now.